.circleci/config.yml

```diff
@@ -286,7 +286,7 @@ jobs:
           name: Test the "marsha" application bootstrapping
           command: |
             bin/ci-bootstrap "marsha"
-            bin/ci-test-service "marsha" "Api Root" "/api/"
+            bin/ci-test-blue-green-service "marsha" "Api Root" "/api/"
```

Comment:
I would have preferred that you update the `bin/ci-test-service` script usage to add the blue-green option instead of creating an almost identical script.

Reply:
:+1: